Superposition/ReferenceImplementation.qs

```diff
@@ -279,8 +279,8 @@ namespace Quantum.Kata.Superposition {
     operation AllStatesWithParitySuperposition_Reference (qs : Qubit[], parity : Int) : Unit is Adj + Ctl {
         // base of recursion: if N = 1, set the qubit to parity
         let N = Length(qs);
-        if (N == 1) {
-            if (parity == 1) {
+        if N ==1 {
```

Comment:
It's a bit more readable to keep the spaces between the operators and the arguments
```suggestion
        if N == 1 {
```